scss/custom.scss

```diff
@@ -280,80 +280,88 @@ $summary-window: #ccd8f075;
     max-width: 98%;
     height: 97%;
     border-radius: 30px;
+    overflow: hidden scroll;
   }
-}
 
-.animation-wrapper {
-  position: absolute;
-  text-align: center;
-  top: 50%;
-  left: 50%;
-  transform: translate(-50%, -50%);
-}
+  .welcome-header {
+    font-family: $header-font-family;
+  }
 
-.loading-bar {
-  width: 6rem;
-  height: 6rem;
-}
+  .main-dashboard-detail {
+    background-color: rgba(204, 216, 240, 0.4588235294);
```

Comment:
Could you consider naming this and grouping it with the other named color variables?

Reply:
Of course, will do.